### DISCUSSION TITLE
Third-party dependencies

### DISCUSSION THREAD

**Discussion by mythitorium** [modrinth/.github#31] [Feature requests]
### Is your feature request related to a problem? Please describe.

Only modrinth projects can be added to a versions' dedicated dependency section.

A solution, linking any third party dependencies in the changelog, does work. However, makes it extremely impractical to get any information regarding said dependencies using the API

### Describe the solution you'd like

Along with the options to get a dependency by version/project id, allow a user to create a dependency listing, by providing the option to enter a link and an input to give the 'third-party' dependency a name. 

These dependencies would which would be listed along side regular dependencies, on website and via the API

**Comment by Andre601** [comment]
What I would find nice is some support for specific third-party Pages if they provide an API or similar for information.

For example, support for SpigotMC resources could be very useful here. People could provide the resource ID and Modrinth would then use the [XenForoResourceManagerAPI](https://github.com/SpigotMC/XenforoResourceManagerAPI) from Spigot to obtain information like the tag line, author and resource icon.

This would really be a nice addition to have and solve a lot of headaches.
